added the crime_reporting

Sharukap opened the pull request:
Added the crime reporting part which is yet to be modularized and the assigning needs to be modified to make it common for all general module submodules

Sharukap committed:
added the crime_reporting

yashodgayashan reviewed (commented):
Please add the crime report as a module. If any doubts please ask me.
Please use underscore method to name the methods.
Please use meaningful names for files as well

achala87 commented:
Please keep crime reports under general module

On Wed, 18 Nov 2020, 09:35 yashodgayashan, <notifications@github.com> wrote:

> *@yashodgayashan* commented on this pull request.
>
> Please add the crime report as a module. If any doubts please ask me.
> Please use underscore method to name the methods.
> Please use meaningful names for files as well
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Reforest-Sri-Lanka/nep/pull/12#pullrequestreview-533089185>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABRZUXGPCNXSGF7FL4MB7O3SQNBXTANCNFSM4TZK5FGQ>
> .
>

Sharukap committed:
modified function names to be meaningful

Sharukap committed:
set up the general and crime report modules

Sharukap commented:
> Please add the crime report as a module. If any doubts please ask me.
> Please use underscore method to name the methods.
> Please use meaningful names for files as well

Added the crime report as a module plus the general module views separately inside the general module
and renamed the functions as requested. Also some of the views and functions will be removed from the crime_reporting when I'm done with the common module of Approval Item. (That should be done by tomorrow)
